lib/Varien/Autoload.php

```diff
@@ -37,7 +37,7 @@ public static function instance()
      */
     public static function register()
     {
-        spl_autoload_register([self::instance(), 'autoload']);
+        spl_autoload_register([self::instance(), 'autoload'], true, true);
```

Comment:
I'm pretty certain it is needed if we want to be able to override Zend classes. I know that the composer autoloader does the same thing for Zend classes anyway, so it would only slow down PSR-4 classes, which I don't think we currently use a lot of. I'll also  investigate this more.